DigitalLearningSolutions.Web/Startup.cs

```diff
@@ -43,18 +42,19 @@ public void ConfigureServices(IServiceCollection services)
                 .AddCookie("Identity.Application", options =>
                 {
                     options.Cookie.Name = ".AspNet.SharedCookie";
+                    options.Cookie.Domain = config.GetValue<string>("SharedCookieDomain");

```

Comment:
This is interesting! When I inspect the cookie on UAT, the domain seems to be set properly, so I was wondering if this was required (e.g. would setting the path be sufficient?)
![image](https://user-images.githubusercontent.com/40016796/116576835-d5b1da00-a907-11eb-8852-d54c45f6c125.png)

Reply:
Oops, just realised this image doesn't have headers! Second try:
![image](https://user-images.githubusercontent.com/40016796/116577038-05f97880-a908-11eb-8907-7a6c44b5bc9d.png)

Reply:
In which case, that hopefully means the changes can be reverted to just manually set the path to be empty. That should make the review easier when I get the change committed.